lighthouse/environment/src/lib.rs

```diff
@@ -256,8 +276,14 @@ impl<E: EthSpec> Environment<E> {
             .open(&path)
             .map_err(|e| format!("Unable to open logfile: {:?}", e))?;
 
-        let drain = Mutex::new(slog_json::Json::default(file)).fuse();
-        let drain = slog_async::Async::new(drain).build();
+        let log_format = log_format.unwrap_or("JSON");
+        let drain = match log_format.to_uppercase().as_str() {
+            "JSON" => {
+                let drain = Mutex::new(slog_json::Json::default(file)).fuse();
+                slog_async::Async::new(drain).build()
```

Comment:
The initial logger here had this code. Wondering if Mutex is really necessary here? I think we could just do 
```rust
let drain = slog_json::Json::default(file)).fuse;
slog_async::Async::new(drain).build()
```

Reply:
Hmm.. it's not clear to me why this is a `Mutex`. Does it compile without the Mutex?

IIRC @kirk-baird added the Mutex originally.

Reply:
Yup, it compiles without it. I asked just to know if there was a specific reason to the Mutex :) I added a new commit that removes the mutex!

Reply:
The tutorial I was following used a mutex, having a look through it now the mutex is unnecessary so happy for it to be removed :)